demos/sslecho/A-SSL-Docs.txt

```diff
@@ -0,0 +1,19 @@
+10-Dec-2021		RCW
```

Comment:
Please remove this line. Version history is managed by git so we don't need to repeat it in the docs.